server/src/services/tooljet_db.service.ts

```diff
@@ -610,7 +610,7 @@ export class TooljetDbService {
 
     if (!internalTable) throw new NotFoundException('Internal table not found: ' + tableName);
 
-    const query = `ALTER TABLE "${internalTable.id}" DROP COLUMN ${column['column_name']}`;
+    const query = `ALTER TABLE "${internalTable.id}" DROP COLUMN "${column['column_name']}"`;
```

Comment:
Can you convert this into a parameterised query since the column name is a param?